### ISSUE TITLE
Institution contact page

### ISSUE THREAD

**Issue by shawnthompson** [ServiceCanada/GCWeb#2]
[Contact](http://servicecanada.github.io/GCWeb/contact/institution-en.html)

Removed the Institution name from the title of the page because it's found in the breadcrumbs. This has been a standard we have been using on Canada.ca.

**Comment by MRicher** [comment]
And when you do a search for a contact of Department X, you will get a lot of title search named just contact.  I think this is a issue.

**Comment by shawnthompson** [comment]
Good point but then the breadcrumbs get repetitive

**Comment by shawnthompson** [comment]
Also on the contact page, why not have the list items with bullets like a regular list?

![capture1](https://cloud.githubusercontent.com/assets/2575266/5856447/4b4f23f8-a211-11e4-9621-71f476189251.JPG)

List on profile page

![capture2](https://cloud.githubusercontent.com/assets/2575266/5856448/4cff900c-a211-11e4-9239-0aafe40457a8.JPG)

**Comment by DK-TBS** [comment]
Yeah, I think we'll have to leave the institution name in the page title.  That way when people gear up a hyperlink to the page, the link text will be descriptive even when taken out of context.  The search results page is a good example of this.  

Agreed on the bullets under the three headings, that's got my vote for sure.  Bullet it up!
